contracts/src/v0.8/dev/vrf/VRFV2PlusWrapper.sol

```diff
@@ -358,10 +358,12 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
   function onTokenTransfer(address _sender, uint256 _amount, bytes calldata _data) external onlyConfiguredNotDisabled {
```

Comment:
I wonder what the gas cost effect would be if this is `bytes memory _data` instead? In my tests on Remix an array access on a `calldata` array is cheaper by about 400 gas.

Reply:
Hmm